src/BaseVault.sol

```diff
@@ -695,24 +630,12 @@ abstract contract BaseVault is IVault, ERC20PermitUpgradeable, AccessControlUpgr
         uint256 totalBaseBalance = _computeTotalAssets();
 
         _getVaultStorage().totalAssets = totalBaseBalance;
+        // solhint-disable-next-line not-rely-on-time
         emit ProcessAccounting(block.timestamp, totalBaseBalance);
     }
 
     function _computeTotalAssets() internal view virtual returns (uint256 totalBaseBalance) {
```

Comment:
let's make this function public

remove the _ i guess if it doesn't break anything